Sources/Kommunicate/Classes/KMConversationViewController.swift

```diff
@@ -639,6 +640,7 @@ extension KMConversationViewController {
         chatBar.clear()
         conversationClosedView.clearFeedback()
         isClosedConversationViewHidden = false
+        awayMessageView.isHidden = true
```

Comment:
Did you check the case of reopen condition ? As of now if conversation is resolved, you are hiding the away message. let say user reopens the conversation, what happens at that time when agent is still away. Will away message be visible or hidden ? @AbhijeetRanjan308

Reply:
yes for that I have added  a condition while the conversation is restarted it set that value to false (in line 472).

Reply:
What if he comes to online before conversation restarted ? @AbhijeetRanjan308

Reply:
after restarting the conversation user need to send a message to receive this away message

Reply:
@Sathyan-KM  it will not so the away message in that condition